src/demangle.cpp

```diff
@@ -117,14 +60,11 @@ void Demangle::cutArguments(char* s) {
 }
 
 char* Demangle::demangle(const char* s, bool full_signature) {
```

Comment:
Note that `Demangle::demangle` is called only for symbols starting with `_Z`.
IIUC, it's not the case for V0 format.

Reply:
the new revision changes that